### ISSUE TITLE
Document the new tutorial on ingesting velocity data and compare it to the model output

### ISSUE THREAD

**Issue by fmaussion** [OGGM/tutorials#34]
The skeleton is here: https://github.com/OGGM/tutorials/blob/master/notebooks/ingest_gridded_data_on_flowlines.ipynb

**Comment by bearecinos** [comment]
Hey I am happy to do this ! is its still an issue

**Comment by fmaussion** [comment]
Yes the tutorial is definitely empty : https://github.com/OGGM/tutorials/blob/master/notebooks/ingest_gridded_data_on_flowlines.ipynb 

;-)

**Comment by minallah** [comment]
Happy to work on this tutorial - I'll be focusing on the HMA region.
